pybaseball/playerid_lookup.py

```diff
@@ -52,8 +52,25 @@ def get_lookup_table(save=False):
     table['name_first'] = table['name_first'].str.lower()
     return table
 
+def name_similarity(last: str, first: str, player_table: pd.DataFrame) -> pd.DataFrame:
+    """Calculates similarity of first and last name provided with all players in player_table
 
-def playerid_lookup(last=None, first=None, player_list=None):
+    Args:
+        last (str): Provided last name
+        first (str): Provided first name
+        player_table (pd.DataFrame): Chadwick player table including names
+
+    Returns:
+        pd.DataFrame: 5 nearest matches from fuzzywuzzy.process
+    """
+    filled_df = player_table.fillna("")
+    chadwick_names = filled_df["name_first"] + " " + filled_df["name_last"]
+    fuzzy_matches = pd.DataFrame(process.extract(f"{first} {last}", chadwick_names, limit=5))
+    matched_names = fuzzy_matches[0].str.split(expand=True)
+    matched_names = matched_names.rename(columns={0:"name_first", 1:"name_last"})
+    return matched_names
+    
+def playerid_lookup(last=None, first=None, player_list=None, search=False):
```

Comment:
I think this needs a doc string, otherwise the cool new feature `search` is obscured

Reply:
Good call, done.